src/ctim/schemas/vocabularies.cljc

```diff
@@ -335,7 +335,12 @@
     "email"
     "imei"
     "imsi"
-    "amp-device"})
+    "amp-device"
```

Comment:
I use the underscore for new values only. For backward compatibility, I kept the dash for old values.

The dash character is not valid for GraphQL enums